### PR TITLE
XDR Mirror Bug - Add More Statuses For Resolved

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/41457

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
@ShahafBenYakir  Waiting on your approval to send to the customer

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 1]
```diff
@@ -2521,7 +2523,8 @@ def handle_incoming_user_unassignment(incident_data):
 
 def handle_incoming_closing_incident(incident_data):
     closing_entry = {}  # type: Dict
-    if incident_data.get('status') in XDR_RESOLVED_STATUS_TO_XSOAR:
+    incident_status = incident_data.get('status')
+    if incident_status in XDR_RESOLVED_STATUS_TO_XSOAR:
```
Comment (ShahafBenYakir) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:2527]: You can use walrus
```suggestion
    if incident_status := incident_data.get('status') in XDR_RESOLVED_STATUS_TO_XSOAR:
```

Comment (tomneeman151293) [reply Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:2527]: Will fix that in the merged PR to master after customer approval it solves the problem

**Comment by tomneeman151293** [comment]
closing as other pr including this changes was merged

**Pull request closed without merging**